pkg/k8s/apis/cilium.io/utils.go

```diff
@@ -27,3 +44,100 @@ func ExtractNamespace(np *metav1.ObjectMeta) string {
 
 	return np.Namespace
 }
+
+// ParseToCiliumRule returns an api.Rule with all the labels parsed into cilium
+// labels.
+func ParseToCiliumRule(namespace, name string, r *api.Rule) *api.Rule {
```

Comment:
Is this function tested anywhere? There is a lot of logic there, would be nice if at least some paths were tested.